DxDispatch/src/dxdispatch/DmlDispatchable.cpp

```diff
@@ -264,13 +563,25 @@ void DmlDispatchable::Initialize()
 
 void DmlDispatchable::Bind(const Bindings& bindings, uint32_t iteration)
 {
-    auto bindingProps = m_operatorCompiled->GetBindingProperties();
+    auto bindingProps = m_compiledOperator->GetBindingProperties();
 
     BindingData inputBindingData = {};
-    FillBindingData(m_desc.bindPoints.inputs, &m_initBindings, &bindings, inputBindingData, false, m_desc.compileType);
-
     BindingData outputBindingData = {};
-    FillBindingData(m_desc.bindPoints.outputs, &m_initBindings, &bindings, outputBindingData, false, m_desc.compileType);
+
+    std::optional<Model::DmlDispatchableDesc::DmlCompileType> compileType = std::nullopt;
+
+    if (std::holds_alternative<Model::DmlDispatchableDesc>(m_desc))
+    {
+        const auto& dmlDesc = std::get<Model::DmlDispatchableDesc>(m_desc);
+        compileType = dmlDesc.compileType;
+    }
+    else if (!std::holds_alternative<Model::DmlSerializedGraphDispatchableDesc>(m_desc))
```

Comment:
Remove this check, it is not needed because the constructor won't accept any other type of dispatchable desc.